armi/cli/clone.py

```diff
@@ -28,24 +28,34 @@ class CloneArmiRunCommandBatch(EntryPoint):
     settingsArgument = "required"
 
     def addOptions(self):
-        for settingName in self.cs.keys():
-            # verbosity and branchVerbosity already have command line options in the default parser
-            # adding them again would result in an error from argparse.
-            if settingName not in ["verbosity", "branchVerbosity"]:
-                self.createOptionFromSetting(settingName)
```

Comment:
For some reason I was getting a weird failure testing the CLI with this being first versus last.

Reply:
I'm indifferent to the position.  Move it if you need.

Strange though.